### ISSUE TITLE
Suddenly getting email notifications - can I turn it off?

### ISSUE THREAD

**Issue by gksmyth** [Bioconductor/legacy.support.bioconductor.org#72]
Historically I have not received email notifications from either Bioconductor Support or from Biostars. My Bioc Supp notification setting is "local messages" while my Biostars setting is "default". Although I haven't been able to find any documentation regarding the meaning of "local messages", I assumed it meant notification only on the website rather than by email. The documentation for "default" says that mail notification occurs only when someone answers one of my questions and, since I hardly ever post a question, that should mean almost no emails.

Two days ago I suddenly started getting notifications from Bioc Supp. I am getting notifications for all activity (questions, answers and comments) on new questions that use one of my tags, even if I haven't answered myself. I am also getting notifications for comments on answers that I posted in the distant past. I tried changing my notification setting from "local messages" to "default", but the emails kept coming.

In this new behaviour intentional? If it is, is there any user account setting that does not involve getting emails? Ironically I'm getting more email now than I did when Bioc Supp was a mailing list, because I used to get one digest a day.

**Comment by lshep** [comment]
I haven't made any changes to the email configuration so it seems odd this behavior is all of a sudden happening.   I can look into it.

**Comment by gksmyth** [comment]
@lshep Thanks for looking into it.

**Comment by gksmyth** [comment]
The emails seem to have stopped now, so I am going to close this issue.

*Edit later*

A few more emails did come when comments were added to the same questions that had generated emails before, but no emails have come for a couple of weeks now.

I don't know whether this is connected, but Bioc support website keeps populating my profile with "Watched tags". I have deleted them a number of times but they keep reappearing after a delay.
